docs/release-notes/highlights-6.3.0.asciidoc

```diff
@@ -1,9 +1,124 @@
+
+[role="screenshot"]
+image::images/management_basic_license.png[]
+
+[float]
+=== Autocomplete and simple syntax
+
+{kib}'s experimental autocomplete and simple syntax features can help you create 
+queries. To turn on these features, click *Options* on the right of the query bar
+and opt in.
+
+Once enabled, just start typing in the query bar, and you'll get suggestions for 
+fields, values for keyword fields, and query operators. For information on the 
+simplified, easier-to-use syntax, see <<kuery-query>>.
+
+
+
+[float]
+=== Return to recently viewed content
+
+{kib} now keeps track of your recently viewed items so that you can easily get 
+back to them.  Here's how:
+
+* Your last five recently viewed items are listed at the top of the Home page. 
+A dropdown menu provides access to views further back in history.
+
+* Your recently selected time ranges are listed in the Recent tab in the time 
+picker.
+
+
+[float]
+=== Smaller grid dimensions for dashboards
+
+{kib} 6.3 gives you more control over panel sizes when creating a dashboard.  
+This gives you better control over the layout.  Here are the changes:
+
+* The grid height of the dashboard was decreased from 100 pixels to 20 pixels.
+
+* The number of columns in the grid was increased from 12 pixels to 48 pixels.
+
+
+[float]
+=== Chained Input Controls
+
+The <<controls, Controls visualization>> now enables you to chain dropdown menus. When 
+chained, the value selected in the first menu filters the terms in the second 
+menu. A chained menu is only enabled when its ancestor controls have values.
+
+[float]
+=== Index Management UI
+
+The new <<managing-indices,Index management>> UI enables you to view index 
+settings, mappings, and statistics and perform management operations. These 
+include refreshing, flushing, clearing the cache, merging segments, and closing 
+or deleting indices. The UI provides a convenient way to perform bulk operations
+on multiple indices.
+
+[role="screenshot"]
+image::images/management_index_bulk.png[Index Management UI]
+
+[float]
+=== Interactive map support in Vega
+
+You can now use <<vega-graph, Vega graphs>> to overlay a  geographic data visualization 
+on a map.  To start a Vega visualization, open the Visualization application and
+click the *+* button. Scroll to the *Others* section and select *Vega*.
+
+[float]
+=== Easily enable monitoring from the UI
+
+In 6.3, the first time you open {kib} monitoring, data collection is disabled,
+and you'll be prompted to turn it on. If you have permission to turn on data 
+collection, and there is data in the cluster, {kib} displays the monitoring
+dashboards. If {kib} can't activate monitoring, you'll get a message with 
+suggestions to try.
+
+[float]
+=== Enhancements to the console
+
+The autocomplete function in the Dev tools console is now expanded to use {kib} 
+and Basic {xpack} APIs. This ensures that autocomplete covers the same use cases
```

Comment:
Hmm I'm not sure `This ensures that autocomplete covers the same use cases as the query bar and other areas of Kibana` is. If anything, we're ensuring that the same API's that are available through Elasticsearch natively now support autocomplete in Console. It's almost a feature parity thing.